### PR TITLE
Update/pre commit

### PR DESCRIPTION
**Pull request by kse3hi**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
 				"python.pythonPath": "/usr/bin/python3",
 				"python.defaultInterpreterPath": "/usr/bin/python3",
 				"[python]": {
-					"editor.formatOnSave": true,
+					"editor.formatOnSave": false,
```
Comment (dennismeister93) [inline .devcontainer/devcontainer.json:20]: why false?

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (kse3hi) [reply .devcontainer/devcontainer.json:20]: It comes from the update on NOTICE_3rd... Auto formatting on save constantly reformats content of updates that leads to enormous difficulties in updating NOTICE...

Comment (dennismeister93) [reply .devcontainer/devcontainer.json:20]: than temporarly turn it off or use the save without formatting option. i am against turning it off

Comment (kse3hi) [reply .devcontainer/devcontainer.json:20]: That exactly what I've been making before... But after 20 times, I found it to be not very efficient...

Comment (kse3hi) [reply .devcontainer/devcontainer.json:20]: In any case... I will revert it...

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**